sktime/forecasting/model_selection/_split.py

```diff
@@ -222,6 +229,67 @@ def _check_window_lengths(
                 raise ValueError(error_msg_for_incompatible_types)
 
 
+def _cutoffs_fh_window_length_types_are_supported(
```

Comment:
please write a docstring - I know these are private, but for sth like this it is very much needed.
For review, future contributors, or just a future version of yourself...

Reply:
added docstrings to all `_check...` functions in `_split.py`